False positives with repeated numbers

I'm using dupword through golangci-lint version 1.50.1, which uses dupword v0.0.7. From looking at [the changes since](https://github.com/Abirdcfly/dupword/compare/v0.0.7...v0.0.8) I don't think the latest release of dupword (v0.0.8) fixes the issue, although I admit I have not actually tested this.

I'm getting false positives on number values in strings:

```
shapefile_test.go:35:42: Duplicate words (5,5) found (dupword)
                        expectedGeom0:      newGeomFromWKT(t, "MULTILINESTRING ((1 5,5 5,5 1,3 3,1 1),(3 2,2 6))"),
                                                              ^
shapefile_test.go:92:42: Duplicate words (60,120) found (dupword)
                        expectedGeom0:      newGeomFromWKT(t, "POLYGON ((-120 60,120 60,120 -60,-120 -60,-120 60),(-60 30,-60 -30,60 -30,60 30,-60 30))"),
```

Note that the duplicated words are `5,5` and `60,120`, which aren't words at all. The lines in their full context are at https://github.com/twpayne/go-shapefile/blob/65f0f2d54aee08071337af8be53dbc55680a056a/shapefile_test.go#L35 and https://github.com/twpayne/go-shapefile/blob/65f0f2d54aee08071337af8be53dbc55680a056a/shapefile_test.go#L92.

Clearly, these are false positives and dupword should not report them.

--- a/dupword.go
+++ b/dupword.go
@@ -289,6 +289,9 @@ func (a *analyzer) Check(raw string) (update string, keyword string, find bool) 
 //	e.g. %s, </div> should not be reported.
 func ExcludeWords(word string) (exclude bool) {
 	firstRune, _ := utf8.DecodeRuneInString(word)
+	if unicode.IsDigit(firstRune) {
+		return true
+	}
 	if unicode.IsPunct(firstRune) {
 		return true
 	}

--- a/dupword_test.go
+++ b/dupword_test.go
@@ -194,7 +194,12 @@ func TestExcludeWords(t *testing.T) {
 		{
 			name:        "number should not exclude",
 			args:        args{word: "3"},
-			wantExclude: false,
+			wantExclude: true,
+		},
+		{
+			name:        "number,number should exclude",
+			args:        args{word: "3,3 3,3"},
+			wantExclude: true,
 		},
 		{
 			name:        "%s should exclude",

